README.md

```diff
@@ -1,28 +1,35 @@
-# Exonum Java Binding
+# Exonum Library for Java
 
 [![Build Status](https://img.shields.io/travis/exonum/exonum-java-binding/master.svg)](https://travis-ci.com/exonum/exonum-java-binding)
-[![Maven Central](https://img.shields.io/maven-central/v/com.exonum.binding/exonum-java-binding-core.svg)](https://mvnrepository.com/artifact/com.exonum.binding/exonum-java-binding-core/latest)
 [![Coverage Status](https://coveralls.io/repos/github/exonum/exonum-java-binding/badge.svg?branch=master)](https://coveralls.io/github/exonum/exonum-java-binding?branch=master)
 [![Join the chat on https://gitter.im/exonum/exonum-java-binding](https://img.shields.io/gitter/room/exonum/exonum-java-binding.svg?label=Chat)](https://gitter.im/exonum/exonum-java-binding)
 
-Exonum Java Binding is a framework for building blockchain applications in Java, 
-powered by Exonum.
+This is a primary repository for Exonum projects in Java.  
+[Exonum][exonum] is an extensible open-source framework for creating blockchain applications.
 
-The project is at alpha stage. Currently you can define a stateful Exonum Service 
-in Java, configure a network of nodes and run the service in it.
+## Projects
+The following projects are available:
+ - [Exonum Java Binding](exonum-java-binding-parent). It is a framework for building blockchain 
+ applications in Java which provides Exonum services runtime environment
+ - [Light Client](exonum-java-light-client). It is a library for interaction with Exonum 
+ on a client site that can be used for creating digital signatures,
```

Comment:
Did you intend to pit `site` instead of `side` here? It also works but to me `client side` seems more common.